tableauserverclient/server/endpoint/tasks_endpoint.py

```diff
@@ -0,0 +1,30 @@
+from .endpoint import Endpoint
+from .. import TaskItem, PaginationItem
+import logging
+
+logger = logging.getLogger('tableau.endpoint.tasks')
+
+
+class Tasks(Endpoint):
+    @property
+    def baseurl(self):
+        return "{0}/sites/{1}/tasks/extract-refreshes".format(self.parent_srv.baseurl,
```

Comment:
No, I'll expand this to handle both before it is released.  This PR will stay around until the server code is done.  Or, maybe I'll make a long-lived branch until it is done.